lib/zuora/objects/usage.rb

```diff
@@ -13,8 +13,8 @@ class Usage < Base
     validates_presence_of :subscription_id, :unless => :subscription_number
     validates_length_of :subscription_number, :maximum => 255, :unless => :subscription_id
 
-    validates_datetime_of :start_date_time, :allow_nil => true
-    validates_datetime_of :end_date_time, :allow_nil => true
+    # validates_datetime_of :start_date_time, :allow_nil => true
+    # validates_datetime_of :end_date_time, :allow_nil => true
```

Comment:
Do we know why these are now commented out?

Reply:
Resolved here [04f281f](https://github.com/Sage/zuora/pull/18/commits/04f281f9a0a0b07961e1b1f4ebd5465beb8c27bc)